app/views/documents/show/_history_tab.html.erb

```diff
@@ -29,10 +29,14 @@
         <% end %>
 
         <% if entry.backdated? %>
-          <% date = format_date(entry.edition.backdated_to) %>
+          <% date = format_date(entry.revision.backdated_to) %>
```

Comment:
Good catch! It'd be good if this bug fix was in a separate commit.

Reply:
This is now in a separate commit with a test file for editing a backdate which checks that there are two timeline entries with different dates.